cmake/libfmt.cmake

```diff
@@ -15,10 +15,10 @@ MACRO(BUNDLE_LIBFMT)
   ExternalProject_Add(
     libfmt
     PREFIX   "${dir}"
-    URL      "https://github.com/fmtlib/fmt/archive/refs/tags/8.0.1.zip"
-    URL_MD5  e77873199e897ca9f780479ad68e25b1
+    URL      "https://github.com/fmtlib/fmt/archive/refs/tags/11.0.0.zip"
+    URL_MD5  f690d14b38d0fa473ea414ecf4e9c1a2
+    CMAKE_ARGS -DFMT_UNICODE=ON
```

Comment:
I'm not exactly sure what you're trying to do here. Run CMake, but neither build or install? Can you explain the effect?

If it is just header-only library, I would think you just need to unpack it.

If you want to do the way  libfmt authors supposed it to be used, simplest way is probably FetchContent, as here https://stackoverflow.com/a/66531890,  which will download, build, install, import config, so below  the only thing needed to use library (no include directories, or path to binaries needed)
```cmake
target_link_libraries(sql fmt::fmt)
```

Reply:
this was enabling the utf flag in the headers:

https://github.com/fmtlib/fmt/blob/master/CMakeLists.txt#L360-L363

Which  becomes redundant once its used in a generator

previous attempts: https://buildbot.mariadb.org/#/grid?branch=refs%2Fpull%2F3385%2Fmerge

Reply:
Oh, I see . This /utf-8 would be enabled automatically if we used target_link_libraries(sql fmt) , but we dont 

Looked up the where /utf-8 error was coming from in previous builds

Author assumes that 
```c++
unsigned char buffer[]="\u00A0"
```
has `buffer` encoded in UTF8. Seem too strong for general case, why would gcc have -finput-charset and -fexec-charset

Anyway, just enabling CONFIGURE would not do anything good for us, and it is good you removed it again